dev-vcs/easygit/easygit-1.7.5.2.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+MY_PN="eg"
+
+DESCRIPTION="Easy GIT (eg), a frontend for git designed for former cvs and svn users"
+HOMEPAGE="https://www.gnome.org/~newren/eg/"
+SRC_URI="https://www.gnome.org/~newren/${MY_PN}/download/${PV}/${MY_PN} -> ${PF}"
+
+LICENSE="GPL-2"
+KEYWORDS="~amd64 ~ppc ~x86"
+SLOT="0"
+IUSE=""
+
+RDEPEND=">=dev-vcs/git-${PV}
+	dev-lang/perl"
+
+src_unpack() {
+	mkdir "${S}" || die
+	cp "${DISTDIR}/${PF}" "${S}/${MY_PN}" || die
+}
+
+src_compile() { :; }
```

Comment:
Do you actually need to do this? Unless I'm mistaken, the default would do nothing here.